src/PhpSpec/Util/VoterInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Util;
+
+interface VoterInterface
```

Comment:
This naming is far too generic. Where you use it elsewhere it's a `nameChecker` which feels closer to a good name.

Find something that describes how this type is _used_, then for the concrete think about _how_ it does it